From abb1776237ef0a18573af02ce867796f9fdf7c5c Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Wed, 5 Oct 2022 18:17:44 +0200
Subject: [PATCH 0245/1141] remoteproc: stm32: Fix info message when pdds
 syscfg is not found

It is not a failure as the pdds sys config is optional.
Change the message to reflect an information, not failure.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: I9300a5be65d1270a295c9f5fb61b047000b45db7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/269209
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 drivers/remoteproc/stm32_rproc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/remoteproc/stm32_rproc.c b/drivers/remoteproc/stm32_rproc.c
index e913d48db6fc..aa29f6422659 100644
--- a/drivers/remoteproc/stm32_rproc.c
+++ b/drivers/remoteproc/stm32_rproc.c
@@ -975,7 +975,7 @@ static int stm32_rproc_parse_dt(struct platform_device *pdev,
 
 	err = stm32_rproc_get_syscon(np, "st,syscfg-pdds", &ddata->pdds);
 	if (err)
-		dev_info(dev, "failed to get pdds\n");
+		dev_info(dev, "pdds sys config not defined\n");
 
 	*auto_boot = of_property_read_bool(np, "st,auto-boot");
 
-- 
2.39.2

